.github/workflows/diff_tests.yml

```diff
@@ -62,13 +62,13 @@ jobs:
       - name: Build old rellic
         shell: bash
         run: |
-          cmake -G Ninja -S old -B rellic-build-old -DVCPKG_ROOT=$GITHUB_WORKSPACE/vcpkg_${{ matrix.image.name }}-${{ matrix.image.tag }}_llvm-${{ matrix.llvm }}_amd64
+          cmake -G Ninja -S old -B rellic-build-old -DCMAKE_TOOLCHAIN_FILE=$GITHUB_WORKSPACE/vcpkg_${{ matrix.image.name }}-${{ matrix.image.tag }}_llvm-${{ matrix.llvm }}_amd64/scripts/buildsystems/vcpkg.cmake
```

Comment:
The test failure is caused by this line because `master` requires `VCPKG_ROOT`. However, once this gets merged it will begin working so I think we can just leave this as is.

Reply:
That's ok, we have a similar issue regarding LLVM14 vs 15